app/src/main/java/ca/bc/gov/bchealth/ui/healthpass/add/FetchVaccineRecordFragment.kt

```diff
@@ -105,6 +103,20 @@ class FetchVaccineRecordFragment : Fragment(R.layout.fragment_fetch_vaccine_reco
         }
     }
 
+    private fun handleError(state: FetchVaccineRecordUiState) {
```

Comment:
This condition should be checked on ViewModel instead of the fragment.
ViewModel should pass the error object with title & message.